Frameworks/CoreGraphics/DWriteFontBinaryDataCollectionLoader.h

```diff
@@ -0,0 +1,64 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#pragma once
+
+#include <COMIncludes.h>
+#import <wrl/implements.h>
+#include <COMIncludes_End.h>
+
+#import <Starboard.h>
+#import <CoreGraphics/DWriteWrapper.h>
+#import <CoreFoundation/CFArray.h>
+#import <CoreFoundation/CFData.h>
+
+#import <vector>
+
+/**
+ * Custom implementation of IDWriteFontCollectionLoader, which creates an IDWriteFontFileEnumerator to create font files
+ * Which are used to create a font collection.  This is expected to be a singleton.
+ */
+class DWriteFontBinaryDataCollectionLoader
+    : public Microsoft::WRL::RuntimeClass<Microsoft::WRL::RuntimeClassFlags<Microsoft::WRL::WinRtClassicComMix>,
+                                          IDWriteFontCollectionLoader> {
+protected:
+    InspectableClass(L"Windows.Bridge.DirectWrite", TrustLevel::BaseTrust);
+
```

Comment:
very true (its not ever going to be used since we aren't registering our runtime classes to be activatable in a RoActivateInstance type scenario) but yes would be great to do.

---
In reply to: [90944936](https://github.com/Microsoft/WinObjC/pull/1504#discussion_r90944936) [](ancestors = 90944936)